tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -0,0 +1,102 @@
+@api @antivirus
+Feature: antivirus
+  As a system administrator and user
+  I want to protect myself and others from known viruses
+  In order to prevent files with viruses from being uploaded
+
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+
+  Scenario Outline: upload a normal file without virus
+    Given using <dav-path-version> DAV path
+    When user "Alice" uploads file "filesForUpload/textfile.txt" to "/normalfile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And as "Alice" file "/normalfile.txt" should exist
+    And the content of file "/normalfile.txt" for user "Alice" should be:
+    """
+    This is a testfile.
+
+    Cheers.
+    """
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+
+    @skipOnRevaMaster
+    Examples:
+      | dav-path-version |
+      | spaces           |
+
+  Scenario Outline: upload a file with virus
+    Given using <dav-path-version> DAV path
+    When user "Alice" uploads file "filesForUpload/filesWithVirus/eicar.com" to "/aFileWithVirus.txt" using the WebDAV API
+    # antivirus service can scan files during post-processing. on demand scanning is currently not available
+    Then the HTTP status code should be "201"
+    And user "Alice" should get a notification with subject "Virus found" and message:
+      | message                                                                             |
+      | Virus found in aFileWithVirus.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Alice" file "/aFileWithVirus.txt" should not exist
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+
+    @skipOnRevaMaster
+    Examples:
+      | dav-path-version |
+      | spaces           |
+
+
+  Scenario Outline: upload a file with virus and a file without virus
```

Comment:
do we have to cover this scenario? Above we covered upload file with virus and without virus separately

Reply:
yes, I think so. Right now the virus scan is done in the post-processing, so if we upload the file with the virus and the normal file at the same time, it's to check that only the file with the virus is deleted.